src/business-logic/services/rundown-timeline-service.ts

```diff
@@ -31,6 +33,13 @@ export class RundownTimelineService implements RundownService {
   ) {}
 
   public async activateRundown(rundownId: string): Promise<void> {
+
+    const basicRundowns: BasicRundown[] = await this.rundownRepository.getBasicRundowns();
+    const activeRundown: BasicRundown | undefined = basicRundowns? basicRundowns.find(rundown => rundown.isActive()) : undefined;
```

Comment:
The ternary is unnecessary :) If the the getBasicRundowns call resolves, then you are guaranteed to have an array of basic rundowns.